backend/lib/spree/backend_configuration.rb

```diff
@@ -6,6 +6,27 @@ module Spree
   class BackendConfiguration < Preferences::Configuration
     preference :locale, :string, default: I18n.default_locale
 
+    # @!attribute [rw] themes
+    #   @return [Hash] A hash containing the themes that are available for the admin panel
+    preference :themes, :hash, default: {
```

Comment:
Not blocking, but if we can use versioned preferences here instead, we'll have the Solidus upgrade process automatically suggest people to use the new default starting from the next version. Did you already consider this?

Reply:
Done!